Sources/Public/iOS/AnimatedSwitch.swift

```diff
@@ -70,7 +70,7 @@ open class AnimatedSwitch: AnimatedControl {
   // MARK: Open
 
   open override func animationDidSet() {
-    updateOnState(isOn: _isOn, animated: true, shouldFireHaptics: false)
+    updateOnState(isOn: _isOn, animated: false, shouldFireHaptics: false)
```

Comment:
Did you mean to change this to `animated: animateUpdateWhenChangingAnimation` instead of the one below?

Reply:
@calda This is purely a slip up! Yes I did. updated in f028aca5e7d992d2b0dab1889fa7c67eadcbb313